streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -339,6 +348,11 @@
                     CommonClientConfigs.DEFAULT_SECURITY_PROTOCOL,
                     Importance.MEDIUM,
                     CommonClientConfigs.SECURITY_PROTOCOL_DOC)
+            .define(DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG,
+                Type.CLASS,
+                LogAndFailExceptionHandler.class.getName(),
```

Comment:
Sounds good to me. Not sure why we started with this annoying `.getName()` in the first place...